config.default.yaml

```diff
@@ -106,6 +106,7 @@ cluster_options:
 
 build_shape_options:
   gadm_layer_id: 1 # GADM level area used for the gadm_shapes. Codes are country-dependent but roughly: 0: country, 1: region/county-like, 2: municipality-like
+  simplify_gadm: false # When true, shape polygons are simplified else no
```

Comment:
By default, let's enable the simplification. This on average speeds up computation and is desirable